study.md

```diff
@@ -17,40 +17,47 @@ language.
 
 Please write the code that would take the following integer `007` and return the
 string `"700"`
-
 ```ruby
-  # your answer here
+007.to_s.reverse
 ```
 
 Please write the code that takes the following array `[23,56,3,7]` sorts it
 then reverses that array returning and modifying the refrence to the original
 array.
 
 ```ruby
-  # your answer here
+ticket = [23, 56, 3, 7]
+
+
+ticket.sort
+
+ticket.reverse
```

Comment:
do these modify the original? Let's see if you have the answer to this in the next question.